code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -98,7 +134,107 @@
 		qdel(O)
 	else
 		return ..()
-		
+
+/mob/living/simple_animal/mouse/attack_ghost(mob/dead/observer/user)
+	user.possess_mouse(src)
```

Comment:
Theres no confirmation dialog it seems - if I constantly click on fire alarms by accident, a ghost is gonna click on a mouse by accident and then there'll be a problem

Reply:
check is done https://github.com/yogstation13/Yogstation/pull/7838/files#diff-9c1590efced1526ca7e5cfdae3a5a19eR941

observer.dm L941